<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StopHeadingChoice</class>
 <widget class="QWidget" name="StopHeadingChoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="labelVehicleLenA">
           <property name="text">
            <string>补偿长度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxVehicleLen">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="labelVehicleWidthA">
           <property name="text">
            <string>补偿宽度</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxVehicleWidth">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QGraphicsView" name="graphicsView"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="labelStopLenA">
           <property name="text">
            <string>偏移长度</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxStopLen">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="valueLabel">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>180</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLabel" name="labelStopWidthA">
           <property name="text">
            <string>偏移宽度</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxStopWidth">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upBtn">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downBtn">
       <property name="minimumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>55</width>
         <height>55</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

